client/src/components/home/slider/slider.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './slider.css';
+import left from '../../../assets/angle-left.svg';
+import right from '../../../assets/angle-right.svg';
+
+// const slider = () => {};
+class slider extends React.Component {
+  state = {
+    counter: 1,
+  };
+
+  componentDidMount() {
+    setInterval(() => {
+      this.plus();
+    }, 2000);
```

Comment:
One alternative to setInterval is `requestAnimationFrame`...which can be better for performance.  No need to change this, just wanted to mention if you haven't seen it before.  :)

https://flaviocopes.com/requestanimationframe/

https://medium.com/@bdc/gain-motion-superpowers-with-requestanimationframe-ecc6d5b0d9a4

Reply:
Its nice method thanks